foursight_core/templates/header.html

```diff
@@ -0,0 +1,327 @@
+<!-- Main body template. Define the base template, <style> and <script> here -->
```

Comment:
This was factored out of base.html. Used by itself in the new info.html.

Reply:
Thanks for the context.